config/outline.go

```diff
@@ -235,12 +240,13 @@ type AccessKey struct {
 	Method           string `json:"method"` // the alias of EncryptionMethod
 }
 
-func (key *AccessKey) ToServer(host string) Server {
+func (key *AccessKey) ToServer(name, host string) Server {
 	method := key.EncryptionMethod
 	if method == "" {
 		method = key.Method
 	}
 	return Server{
+		Name:     fmt.Sprintf("%s - %s", name, key.Name),
```

Comment:
I suggest that Name should be key.Name and the renaming can be done in function outline.GetServers, so that conf.ToServers can represent its meaning that "convert ShadowboxConfig to servers" and the variable name can be more explanatory.